lib/CONST.jsx

```diff
@@ -348,6 +348,10 @@ export const CONST = {
             DEV: 'https://s3.amazonaws.com/dev-receipts-expensify-com/',
             PROD: 'https://s3.amazonaws.com/receipts.expensify.com/'
         },
+        RECEIPTS: {
```

Comment:
Can't comment above... is `BUCKET_RECEIPT` unused now?

Reply:
I think so, but I want to deploy the whole thing before removing it (specifically in case it needs to be rolled back) 

Issue here: https://github.com/Expensify/Expensify/issues/191720